homeworks/README.md

```diff
@@ -0,0 +1,41 @@
+# Installation instructions
+To complete the assignments, you need a working python installation with jupyter and the following libraries: numpy, scipy, matplotlib, 

```

Comment:
Insert a line break above (here and everywhere else below, separate titles from text with an empty line).

Reply:
`python` -> `Python`
`jupyter` -> `Jupyter`